crates/moonbuild/src/gen/gen_runtest.rs

```diff
@@ -963,7 +955,7 @@ fn gen_generate_test_driver_command(
     .arg(&moonbuild_opt.source_dir.display().to_string())
     .arg("--target-dir")
     .arg(&moonbuild_opt.target_dir.display().to_string())
-    .args(test_filter_command)
+    .args(["--package", &item.package_name])
```

Comment:
also need to clean up file and index field in `struct GeneratedTestDriverSubcommand`